components/pricing_modal/starter_disclaimer_cta.tsx

```diff
@@ -48,12 +48,13 @@ function StarterDisclaimerCTA() {
                 },
                 description: {
                     id: t('workspace_limits.modals.informational.description.starterLimits'),
-                    defaultMessage: 'Cloud starter is restricted to {messages} message history, {storage}GB file storage, {integrations} apps, and {boards} board cards.',
+                    defaultMessage: '{planName} is restricted to {messages} message history, {storage} file storage, {boards} board cards, and {integrations} integrations.',
```

Comment:
The `{planName}` still seems to be putting it in lower case. Any chance we can capitalized?